internal/jobcontainers/path.go

```diff
@@ -73,8 +72,10 @@ func getApplicationName(commandLine, workingDirectory, pathEnv string) (string,
 		result     string
 	)
 
-	// Clean the path, to get rid of any . elements
-	commandLine = filepath.Clean(commandLine)
+	// Handle a relative path with .
+	if len(commandLine) >= 3 && commandLine[0:2] == "./" || commandLine[0:2] == ".\\" {
```

Comment:
`&&` has higher precedence than `||`, so this probably isn't doing what you intended.

Reply:
Oop forgot the parens around the second half